src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -98,8 +98,12 @@ private static void addFilterLine(final VCFFilterHeaderLine line) {
         addInfoLine(new VCFInfoHeaderLine(LOW_MQ_KEY, 3, VCFHeaderLineType.Float, "3-tuple: <fraction of reads with MQ=0>,<fraction of reads with MQ<=10>,<total number of reads>"));
```

Comment:
same here - some beagle stuff changed, rest is the same (some are unported eg FractionInformativeReads)